driver/csiplugin/gpfs_util.go

```diff
@@ -199,6 +238,10 @@ func getScaleVolumeOptions(volOptions map[string]string) (*scaleVolume, error) {
 		}
 	}
 
```

Comment:
should we add check for empty NodeClass like done for other ?

Reply:
Empty nodeclass has the same effect as no nodeclass specified. This check is not needed.